contents/handbook/growth/marketing/index.md

```diff
@@ -4,49 +4,84 @@ sidebar: Handbook
 showTitle: true
 ---
 
-Regularly publishing useful content helps us:
+We're in the process of building a cross-functional marketing team to initially convert technical teams and individuals developers at startups, thereafter expanding to other business functions and scale-ups and enterprises.
 
-* Build awareness of PostHog in the developer community
-* Convert people who are aware of us into users
-* Convert users into power users
+## Marketing Functions
 
-## Content Personas
+Our marketing functions will include product marketing, content marketing, community, paid and operations but as a team we work closely with the product team, engineering and specifically growth engineering to improve AARRR, support and sales.
 
-We tailor our content to specific audiences.
+### Product Marketing
 
-### Raising awareness
+Product marketing is the foundation upon which other marketing efforts are built. Product marketing consists of the following activities:
 
-Many developers have no idea who we are, and do not have a current need for PostHog at this time. However, they may develop a need in future. These people are interested in broader content:
+- Market research to identify targetable industries and use cases
+- Competitor research
+- Working with our product team to develop a competitive product positioning
+- Developing and maintaining buyer personas
+- Creating product messaging for our website to address our buyer personas
+- Creating gated marketing content and sales collateral: reports, white papers etc
+- Measuring the success of content
 
-* Syndicating content about how to make your product successful - being a pundit.
-  * New and useful open source libraries to build / understand user behavior
-  * Syndicating content about talking to users and how to help engineers become better at product-thinking
-  * Running an engineering team that ships
-  * Big new releases in popular libraries that people use to ship with
-  * Useful new products that are released
-* 1st hand content about building an open source company
-  * Building in the open
+**Hiring**: We're hiring an intermediate technical product marketer **soon** to address developers/startups and will expand the team when we're reaching our capacity (approximately 4-8 months).
 
-There are also developers that have no idea who we are, but with a specific need that we could solve.
+### Content Marketing
 
-**Suitable Content**
+Content marketers are our story tellers. They excel at creating narrative driven content most commonly found on our blog but can also be in other formats. Using our foundational research and messaging, content marketers have the following responsibilities:
 
-* Tutorials explaining the use cases of our product
+- Researching and pitching innovative ideas
+- Creating articles, interviews, case studies, infographics and ebooks
```

Comment:
It's a nit, but I think infographics/ebooks/ perhaps gated content (?) sound dated as a tactic. For the sake of appealing to prospective candidates perhaps we could remove.

Reply:
It's still highly relevant content critical to a gated content strategy and most marketers will expect to see this as part of their role. Do you feel it will be unappealing to marketers? There are definitely boring/dull implementations of these but if we do them we will strive for something innovative.

Reply:
Sure, I'm a bit of this [school of thought](https://www.drift.com/blog/year-without-forms/), but this is a minor debate to settle another time, happy to merge as is :)

Reply:
Good read, most of that aligns with my views too. 

A question/challenge for us to think about, how can we do some gated content in a meaningful way (it makes sense to give my email because I want to find out more) that gives people extra value beyond our non-gated "free" content?